openmdao_book/other_useful_docs/api_translation.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<p>I know this is correct but maybe we should include some extra text for some of these to point out the differences? It took me a long time to notice what was different in the 3.0 version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>Adding notes for the smaller changes</p>